_data/sidebars/home_sidebar.yml

```diff
@@ -56,12 +70,22 @@
           url: /scheduler/kubernetes/failover.html
         - title: Run Hyper Converged
           url: /scheduler/kubernetes/scheduler-convergence.html
+      - title: Operate and Maintain
```

Comment:
I think if we move k8s content to Operate and Maintain section, then we need to group it under a Kubernetes directory in that section.

Reply:
The existing "Operate and Maintain" page doesn't have an orchestration category. So it'll be odd if I create a Kubernetes directory there. I wanted all k8s related docs to stay under one tree. Hence created a dedicated "Operate and Maintain" for k8s itself.

The idea is people who use Kubernetes, will only be concerned with the Kubernetes section of the docs instead of the overall structure of the docs page.

Thoughts?

Reply:
ah I see.  But we'll still keep the other Operate and Maintain section as well?

Reply:
Yes, the other Operation and Maintain contains generic instructions, not tied to a specific orchestrator. I think over time, the docs will evolve to a per-orchestrator experience since generic instructions are usually not helpful.